e2e/support/commands/overwrites/log.js

```diff
@@ -1,14 +1,40 @@
-Cypress.Commands.overwrite("log", (originalFn, message) => {
-  Cypress.log({
-    displayName: `--- ${window.logCalls}. ${message} ---`,
-    name: `--- ${window.logCalls}. ${message} ---`,
+Cypress.Commands.overwrite("log", (originalFn, text) => {
+  const logConfig = {
+    displayName: `${window.logCalls}. ${text}`,
+    name: "log",
     message: "",
-  });
+  };
 
+  appendStyleIfItDoesntExists(logConfig);
+  Cypress.log(logConfig);
   window.logCalls++;
 });
 
 // We want to reset the log counter for every new test (do not remove from this file)
 beforeEach(() => {
   window.logCalls = 1;
 });
+
+function appendStyleIfItDoesntExists(logConfig) {
```

Comment:
I improve the performance a bit by adding an id to the style tag, so we don't need to query all styles again + clean up some syntax.

Here's the proposed function body
```ts
  const headHTML = window.top.document.head;
  const customStyleExists =
    window.top.document.getElementById("customLogStyle");

  if (!customStyleExists) {
    const style = document.createElement("style");

    style.id = "customLogStyle";
    style.textContent = `
      .command-name-${logConfig.name} .command-pin-target{
        color: #ffffff !important;
        background-color: #7f43c9 !important;
        font-weight: bold !important;
      }
    `;
    style.type = "text/css";
    style.dataset.customLog = true;

    headHTML.append(style);
  }
```

Reply:
Thanks.
I'll adjust it slightly to DRY it and will apply.